pkg/integrations/oracledb_exporter/oracledb_exporter.go

```diff
@@ -0,0 +1,114 @@
+package oracledbexporter
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	oe "github.com/observiq/oracledb_exporter/collector"
```

Comment:
is there any reason to use here a custom fork instead of the original exporter?

Reply:
The custom fork is needed as the original doesn't package in a compatible way with the Agent. ObservIQ will be maintaining the fork to stay up to date

Reply:
I'd be more inclined if those changes can be merged upstream soon and we use the original exporter. It's already happening that we use custom forks for other integrations and it's getting hard to keep them up to date.